### ISSUE TITLE
How does one make an expansion? (this is a question, not a bug)

### ISSUE THREAD

**Issue by essenjey01** [LemonyPancakes/Origins-Bukkit#49]
### Minecraft Server's Version and Software

Paper version git-Paper-81
minecraft version: 1.19

### Plugin's Version

2.3.7

### Bug Description

I was looking at the plugins' folder and i saw this "expansions" forlder inside it. How do I make one? What things does it allow me to change?

### To Reproduce

This is a question, not a bug

**Comment by LiamKrenn** [comment]
did you find something?

**Comment by Blackilykat** [comment]
No, I ended up directly editing the plugin. Not great if you're planning to
publish it but I only had to put it in my own server so it worked fine for
me.
